mlir/test/CMakeLists.txt

```diff
@@ -240,6 +243,65 @@ add_custom_target(check-mlir-build-only
 )
 set_target_properties(check-mlir-build-only PROPERTIES FOLDER "MLIR/Tests")
 
+get_property(LLVM_EXPORTS GLOBAL PROPERTY LLVM_EXPORTS)
+get_property(MLIR_EXPORTS GLOBAL PROPERTY MLIR_EXPORTS)
+if(CLANG_ENABLE_CIR)
+  list(APPEND MLIR_EXPORTS cir-opt cir-translate cir-lsp-server)
+endif()
+add_custom_target(
+  install-mlir-standalone-test-prefix
+  DEPENDS ${MLIR_TEST_DEPENDS} ${MLIR_EXPORTS} FileCheck count not
+  COMMAND ${CMAKE_COMMAND} -E make_directory "${MLIR_STANDALONE_INSTALL_TEST_PREFIX}"
+  COMMAND "${CMAKE_COMMAND}"
+          -DCMAKE_INSTALL_PREFIX="${MLIR_STANDALONE_INSTALL_TEST_PREFIX}"
+          -P "${MLIR_BINARY_DIR}/cmake_install.cmake"
```

Comment:
I would think this command to be already pretty encompassing  in itself, why are the other ones all needed below? (`mlir-headers` for example?)